Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -94,7 +94,7 @@ void SettingsWindow::AddCategoryToList(const QString& title, const std::string&
 void SettingsWindow::MakeCategoryList()
 {
   m_categories = new QListWidget;
-  m_categories->setMaximumWidth(175);
+  m_categories->setFixedWidth(125);
```

Comment:
Looks like there's two things contributing to that space: the actual height of the icons (32, not 28), and some extra padding added by QListWidget (I'm guessing, because the total pixel height without the sizeHint isn't 32) (I'm sure they have some good reason for this).

Since the vertical size of this list is much smaller than the settings panes, I think it's probably best to let Qt figure out the size/padding rather than set the sizeHint explicitly?